node/src/neighborhood/mod.rs

```diff
@@ -4057,13 +4061,9 @@ mod tests {
             bc_from_nc_plus(neighborhood_config, make_wallet("earning"), None, "test");
         let mut subject = Neighborhood::new(main_cryptde(), &bootstrap_config);
 
-        subject.process_acceptance_result(
-            GossipAcceptanceResult::Absorbed,
-            "new-IP Node",
-            27,
-        );
+        subject.process_acceptance_result(GossipAcceptanceResult::Absorbed, "new-IP Node", 1);
 
-        TestLogHandler::new().exists_log_containing("27-Node Gossip from new-IP Node absorbed");
+        TestLogHandler::new().exists_log_containing("1-Node Gossip from new-IP Node absorbed");
```

Comment:
This log isn't strong enough. It lacks the log level and also the name of the logger which should correspond to the name of this test.

Reply:
Fair enough.